.Gruntfile.babel.js

```diff
@@ -266,7 +265,15 @@ module.exports = (grunt) => {
       plugins: [
         alias({
           // https://vuejs.org/v2/guide/installation.html#Standalone-vs-Runtime-only-Build
-          vue: path.resolve('./node_modules/vue/dist/vue.common.js')
+          vue: path.resolve('./node_modules/vue/dist/vue.common.js'),
+          '~': path.resolve('./')
+        }),
+        aliasModuleSpecifiers({
+          '@views': './frontend/views',
+          '@components': './frontend/views/components',
+          '@containers': './frontend/views/containers',
+          '@utils': './frontend/utils',
+          '@assets': './frontend/assets'
```

Comment:
Great job on this :+1:

Reply:
I am curious @pieer, did you try to do this with just the `alias` plugin (like you did for `~`)? Is there a reason we needed a new plugin?

Reply:
I played around with it and figured out how to do it with just `rollup-plugin-alias`:

```js
        alias({
          resolve: ['.vue', '.js'],
          // https://vuejs.org/v2/guide/installation.html#Standalone-vs-Runtime-only-Build
          vue: path.resolve('./node_modules/vue/dist/vue.common.js'),
          '~': path.resolve('./'),
          '@views': path.resolve('./frontend/views'),
          '@components': path.resolve('./frontend/views/components'),
          '@containers': path.resolve('./frontend/views/containers'),
          '@utils': path.resolve('./frontend/utils'),
          '@assets': path.resolve('./frontend/assets')
        }),
```

The trick was to add `          resolve: ['.vue', '.js'],`

So `rollup-plugin-alias-module-specifiers` does not need to be added.